.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,71 @@
+---
+name: Bug report
+about: Report bugs in the software
+title: ''
+labels: bug, to be assigned
+assignees: ''
+
+---
+
+<!-- Provide a general summary of the issue in the title above.
+
+When adding program output or code snippets in the issue, use code blocks with
+appropriate syntax highlighting. -->
+
+#### Description
+<!-- Describe the issue briefly. -->
+
+##### Relates to
+<!-- Add links to any related issue or other references here. -->
+
+#### Expected behaviour
+<!-- Describe what is expected to happen. -->
+
+#### Current behaviour
+<!-- Describe what is actually happening. -->
+
+#### Reproducing the behaviour
+<!-- Describe a set of steps to reproduce the bug. -->
+
+1. 
+2. 
+3.  
+
+##### Software version
+<!-- Tell us about the version of software your using. You can also provide the
+git commit id, if you had built it from the source. -->
```

Comment:
```suggestion
<!-- Tell us about the version of software you're using. You can also provide the
git commit id, if you have built it from source. -->
```
Does this relate to the version of the Node? I think in the early development stages, we should request users to always use the latest `develop` commit. Then we should change this notice to "Please make sure that you have used the latest version of dst-go before submitting a bug report." or "latest version of the develop branch".

Reply:
Addressed suggested changed in this commit - 1c2dcf9403c97c826b7fa701b7398c9a3a379363.

Git commit id is a better reference that latest commit on `develop` branch. Because the latter will change when each pull request is merged while the former is constant.

In practice, the git commit id, in most cases will represent the tip of `develop` branch at the time of creating  the issue.